rnd/autogpt_builder/src/components/admin/marketplace/AdminFeaturedAgentsControl.tsx

```diff
@@ -55,9 +56,15 @@ export default async function AdminFeaturedAgentsControl({
             component: <Button>Remove</Button>,
             action: async (rows) => {
               "use server";
-              const all = rows.map((row) => removeFeaturedAgent(row.id));
-              await Promise.all(all);
-              revalidatePath("/marketplace");
+              return await Sentry.withServerActionInstrumentation(
```

Comment:
LGTM, but I think this is definitely worth a follow-up:
The changes wrapping sentry here can be a wrapper function.

Reply:
100% agree